lib/jobs/ipmi-sdr-alert-job.js

```diff
@@ -45,7 +45,45 @@ function ipmiSdrPollerAlertJobFactory(
     }
     util.inherits(IpmiSdrPollerAlertJob, PollerAlertJob);
 
+    IpmiSdrPollerAlertJob.prototype._formatSdrAlert = function _formatSdrAlert(data) {
+        /*
+         * data format:
+           [ { user: 'admin',
+               host: '192.168.188.91',
+               node: '5858c7f7f6f0ce7d08a7298b',
+               inCondition: true,
+               workItemId: '5858c89bca4349fa0409ba8d',
+               pollerName: 'sdr',
+               reading: [Object] } ]
+         * expected event format:
+           [{ type: 'polleralert',
+              action: 'sdr.updated',
+              typeId: '5858c89bca4349fa0409ba8d',
+              nodeId: '5858c7f7f6f0ce7d08a7298b',
+              data: {
+                 { user: 'admin',
+                   host: '192.168.188.91',
+                   inCondition: true,
+                   reading: [Object] }
+              }} ]
+         */
+        assert.array(data, 'data');
+
+        return _.transform(data, function(result, item){
+            var tmp = {};
+            tmp.type = 'polleralert';
+            tmp.action = 'sdr.updated';
+            tmp.typeId = item.workItemId;
+            tmp.nodeId = item.node;
+            tmp.severity = "information";
```

Comment:
does original sdr report didn't indicated it's 'critical' or 'warning' like sel report?